### ISSUE TITLE
[Feature Request] "Card is in preconstructed decks:" should show all decks regardles of printing

### ISSUE THREAD

**Issue by SuppenNudel** [taw/magic-search-engine#156]
I noticed on mtg.wtf that when looking for Chandra, Torch of Defiance the list of decks the card is printed in under " Card is in preconstructed decks:" is different depending on the printing. The Kaladesh printing shows Hazoret Aggro (Rivals of Ixalan Challenger Deck) while the challenger deck version shows Mono-Red Burn (Pioneer Challenger Decks 2021 Pioneer Challenger Deck).
Technically that is correct. But I'd like it to show all the decks the card regardles of version is printed in. Would you be able to add that?

**Comment by taw** [comment]
Yeah, you're probably right. It might be more useful to list all decks.

I'll need to think what to do with cards that are in too many decks (like basics).

**Comment by taw** [comment]
A few years later, [but this features works now](https://mtg.wtf/card/availability/prm/70934). You need to click "View availability of all printings" to see the rest, as that list is often too long to show by default.
